CITATION.cff

```diff
@@ -0,0 +1,20 @@
+# YAML 1.2
+---
+abstract: "We describes the steps necessary to port OpenL3 audio embeddings1(Cramer et al., 2019)to PyTorch2.  We do regression tests to demonstate and elucidate differences between themodels.  The largest mean absolute error (<1e−3) occurs when using mel-spectrogramson high-pitched frequency.Keywords:Audio Embedding, pytorch, Mel Spectrogram, Spectrogram"
```

Comment:
Can you make this on several lines? Also, typos. 

No one calls things PyTorch2. Please use the name and capitalization that is standard

Reply:
Okay